repos/system_upgrade/el7toel8/actors/authselectcheck/actor.py

```diff
@@ -83,14 +83,24 @@ def produce_authconfig_configuration(self, model, command):
             )
         )
 
-        report_generic(
-            title='Authselect will be used to configure PAM and nsswitch.conf.',
-            summary='There is a new tool called authselect in RHEL8 that '
-                    'replaced authconfig. The upgrade process detected '
-                    'that authconfig was used to generate current '
-                    'configuration and it will automatically convert it '
-                    'to authselect. Authselect call is: {}'.format(command)
-        )
+        create_report([
```

Comment:
as discussed offline, synchronize using either list or objects as argument for create_report

Reply:
fixed